app/router.ts

```diff
@@ -24,19 +24,22 @@ const Router = EmberRouter.extend({
             }
         }
 
-        this.set('readyBlocker', this.get('ready').getBlocker());
+        if (this.readyBlocker) {
```

Comment:
Could by DRYer:
```ts
if (!this.readyBlocker || this.readyBlocker.isDone()) {
    this.readyBlocker = this.get('ready').getBlocker();
}
```

Also do services still require `.get()`?

Reply:
lol, yep, my brain is not awake yet this morning

Reply:
And services need `.get()` here only because the typedefs say they need to.  We're not using `@service` here so it uses the Ember typedefs, which say service injection is a computed (which is true), but the typedefs still say you need to use `.get()` for a computed (because you do in Ember < 3.1).